src/spark/cluster.py

```diff
@@ -119,8 +120,18 @@ def create_cluster(
         worker_memory: Memory per worker (e.g., "10GiB")
         master_cores: CPU cores for master
         master_memory: Memory for master (e.g., "10GiB")
+        force: Skip confirmation prompt if True
     """
-    # TODO - inform user that a new cluster will overwrite the existing cluster
+
+    print("WARNING: Creating a new Spark cluster will terminate your existing cluster.")
+    print("All active Spark sessions and computations will be lost.")
```

Comment:
This will always print on notebook startup IIUC. Is that what you want?

Reply:
you mean when the method is being called? I think that's fine.

Reply:
e.g. from https://github.com/kbase/cdm-jupyterhub/pull/175/files#diff-5c1372251bcdee275758e2d3511fa8c50bd6c52a792739bd5d93d07fac46d6adR70

So when the notebook starts, IIRC, it'll print those two lines

Reply:
👍  I see. yea. moved print statements inside of `if not force`.